pkg/scaffold/olm-catalog/csv.go

```diff
@@ -111,12 +130,18 @@ func (s *CSV) CustomRender() ([]byte, error) {
 	return yaml.Marshal(&cu)
 }
 
-func getCurrentCSVIfExists(csvPath string) (*olmapiv1alpha1.ClusterServiceVersion, bool, error) {
-	if _, err := os.Stat(csvPath); err != nil && os.IsNotExist(err) {
+func (s *CSV) getBaseCSVIfExists() (*olmapiv1alpha1.ClusterServiceVersion, bool, error) {
+	lowerProjName := strings.ToLower(s.ProjectName)
+	name := lowerProjName + CSVYamlFileExt
+	fromCSV := filepath.Join(scaffold.OLMCatalogDir, name)
+	return getCSVFromFSIfExists(s.getFS(), fromCSV)
+}
+
+func getCSVFromFSIfExists(fs afero.Fs, path string) (*olmapiv1alpha1.ClusterServiceVersion, bool, error) {
+	if _, err := fs.Stat(path); err != nil && os.IsNotExist(err) {
 		return nil, false, nil
 	}
-
-	csvBytes, err := ioutil.ReadFile(csvPath)
+	csvBytes, err := afero.ReadFile(fs, path)
```

Comment:
Will `afero.ReadFile(fs, path)` return a file not found error in the same way that `fs.Stat(path)` does above? If so, can we remove the `fs.Stat` call and check for that error here instead?